contracts/staking/contracts/src/interfaces/IStructs.sol

```diff
@@ -53,28 +53,19 @@ interface IStructs {
     /// Note that these balances may be stale if the current epoch
     /// is greater than `currentEpoch`.
     /// Always load this struct using _loadSyncedBalance or _loadUnsyncedBalance.
-    /// @param isInitialized
     /// @param currentEpoch the current epoch
     /// @param currentEpochBalance balance in the current epoch.
     /// @param nextEpochBalance balance in `currentEpoch+1`.
     struct StoredBalance {
-        bool isInitialized;
         uint32 currentEpoch;
```

Comment:
Optional, but could up `currentEpoch` to `uint64` now that `isInitialized` is no longer needed. It'll still fit into a word.

Reply:
Probably worth it.